src/main/java/org/myrobotlab/service/meta/ArduinoMeta.java

```diff
@@ -20,29 +18,25 @@
    * to be constructed. It has description, categories, dependencies, and peer
    * definitions.
    * 
-   * @param name
+   * @param type
```

Comment:
this isn't a param anymore...

Reply:
yah ... javadoc will have a field day before the elves are happy